load-code/pe/parse-pe-export-data.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: parse PE export data
+    namespace: load-code/pe
+    author: "@Ana06"
+    scope: function
+    references:
+      - Practical Malware Analysis, Chapter 19
+    examples:
+      - E4C33AC3638EEF68311F8AC0D72483C7:0x401390
+  features:
+    - and:
+      - offset: 0x3C = IMAGE_DOS_HEADER.e_lfanew
+      - or:
+        - offset/x32: 0x78 = IMAGE_NT_HEADERS32.OptionalHeader.DataDirectory.VirtualAddress
+        - offset/x64: 0x88 = IMAGE_NT_HEADERS64.OptionalHeader.DataDirectory.VirtualAddress
+      - offset: 0x18 = IMAGE_EXPORT_DIRECTORY.NumberOfNames
+      - offset: 0x20 = IMAGE_EXPORT_DIRECTORY.AddressOfNames
+      - offset: 0x24 = IMAGE_EXPORT_DIRECTORY.AddressOfNameOrdinals
+      - offset: 0x1C = IMAGE_EXPORT_DIRECTORY.AddressOfFunctions
```

Comment:
For large functions I could imagine there to be false-positives. Maybe grouping some into basic block subscopes could help.
I'm only guessing though since we don't have https://github.com/fireeye/capa/issues/424 yet.

Reply:
This is most likely implemented with an `if-else`, so I don't think we can split it in basic blocks:

![Screenshot 2021-03-25 at 19 14 53](https://user-images.githubusercontent.com/16052290/112522833-63952500-8d9e-11eb-8189-b80406be2e14.png)